### PR TITLE
Fixture fix

### PR DESCRIPTION
**Pull request by phlax**
Atm some Stores are being created with a `/` in their name, and the wrong parent in tests.

This PR addresses that.

### CODE REVIEW

File: pytest_pootle/fixtures/import_export_fixtures.py [hunk 1]
```diff
@@ -65,6 +65,9 @@ def _cleanup():
 @pytest.fixture
 def en_tutorial_ts(settings, english_tutorial, ts_directory):
     """Require the en/tutorial/tutorial.ts store."""
+
+    english_tutorial.project.localfiletype = "ts"
+    english_tutorial.project.save()
```
Comment (unho) [inline pytest_pootle/fixtures/import_export_fixtures.py]: I think it is better to instead use a different project instead of hijacking tutorial.

Comment (phlax) [inline pytest_pootle/fixtures/import_export_fixtures.py]: you are probs right - but that is how its done currently - and im reluctant to change as tbh it needs a bit of a rewrite.

This change only allows ts in that Project - which should have been happening anyway. Im going to raise a related but that Store.create should check for localfiletype (soon to be filetypes)

Comment (phlax) [inline pytest_pootle/fixtures/import_export_fixtures.py]: opened https://github.com/translate/pootle/issues/4988

**Comment by unho** [comment]
lgtm.

**Pull request merged**